src/Mars-Gtk/GtkWidget.class.st

```diff
@@ -462,6 +462,13 @@ GtkWidget >> getPreferredSizeMin: minimum_size natural: natural_size [
 		GtkRequisition *natural_size))
 ]
 
+{ #category : #geometry }
+GtkWidget >> getScaleFactor [
```

Comment:
please, rename this method to just `#scaleFactor`

Reply:
done!